content/blog/announcing-hpe-swarm-learning-2-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing HPE Swarm Learning 2.0.0
+date: 2023-05-11T04:47:46.996Z
+featuredBlog: true
+author: HPE Swarm Learning Team
+authorimage: /img/Avatar1.svg
+disable: false
+---
+<!--StartFragment-->
```

Comment:
Please remove the "\<!--StartFragment-->"
This might be the result of a copy/paste from MS_Word without using "Paste as Plain Text".

In the CMS editor, please toggle on to "Markdown", you will then see the "\<!--StartFragment-->".

Reply:
Removed